deploy/docker-compose/compose/values.yaml

```diff
@@ -31,15 +32,15 @@ networkMode: bridge
 
 zookeeper:
   size: 3
-  image: apachebookkeeper/bookkeeper-current:latest
+  image: zookeeper:3.8.0
```

Comment:
Hi @hangc0276 ,
I changed the image to the official Zookeeper because I find it more intuitive to run a cluster that consists of BookKeepers and Zookeepers original images. Since this is a sample/demo, treating BookKeeper and Zookeeper as separate applications made it easier for me to understand how things work.